packages/app/src/components/nominationInfo/healthProviderDetail.js

```diff
@@ -1,69 +1,51 @@
 import React, { useContext } from 'react';
-import styles from "./styles.module.css";
-import { Link, useParams } from "react-router-dom";
-import { SearchResultDataContext } from '../../utils/context/SearchResultsContext';
-import { NominationsDataContext } from '../../utils/context/NominationsContext';
-
+import styles from './styles.module.css';
+import { useParams } from 'react-router-dom';
 
 function HealthProviderDetail(props) {
-      
-    const [SearchResultData, setSearchResultData] = useContext(
-      SearchResultDataContext
-      );
-      
-    const [NominationsData, setNominationsData] = useContext(
-      NominationsDataContext
-      );
-        
-    const findSearchResults = (searchTerm) => {
-          let filteredNoms = []
-          if(NominationsData && searchTerm !== undefined) {
-            filteredNoms = NominationsData.filter((nomination) => {
-              return [
-                nomination.providerName,
-                nomination.patientName,
-                nomination.nominationName,
-                nomination.representativeName,
-              ].some((nom) => nom.includes(searchTerm));
-            });
-            setSearchResultData(filteredNoms);
-          } 
-        }
+  function openNewBackgroundTab(val) {
+    var a = document.createElement('a');
+    a.href = `/searchhealthprovider/${val}`;
+    var evt = document.createEvent('MouseEvents');
+    evt.initMouseEvent('click', true, true, window, 0, 0, 0, 0, 0, true, false, false, false, 0, null);
+    a.dispatchEvent(evt);
+  }
+
+  const { id } = useParams();
 
-    const { id } = useParams();
-        
-    const handleSubmit =  (val) => {
-      if (val) {
-        findSearchResults(val);
-        window.open(`/nomination/${id}`, '_blank', 'noopener,noreferrer');
-        return <Link to={`${process.env.APP_URL}/searchresults`} target="_blank" />
-      }
+  const handleSubmit = (val) => {
+    if (val) {
+      window.open(`/nomination/${id}`, '_blank');
+      openNewBackgroundTab(val);
     }
+  };
 
-    const Switch = SearchResultData.length < 1 ? ( "/" ) : ( "/searchresults" );
+  return (
+    <div className={styles.main}>
+      <div className={styles.header}>
+        <label className={styles.title}>{props.title}</label>
+      </div>
+      <div className={[styles.content, props.gridContent && styles['grid-container']].join(' ')}>
+        {props.fields.map((obj, index) =>
+          obj.label === 'Provider Name' ? (
+            <div key={index} className={obj.label === '' ? styles.mobileHide : ''}>
+              <label className={styles.label}>{obj.label}</label>
+              <a>
+                <span className={styles.value} onClick={() => handleSubmit(obj.value)} key={index}>
+                  {String(obj.value)}
```

Comment:
I could be wrong but I believe you can omit the call to `String()` here as well as the one below.